dedibox-console/my-account/how-to/resolve-abuse.mdx

```diff
@@ -7,31 +7,27 @@ content:
   paragraph: This page explains how to resolve Abuse tickets.
 tags: resolve abuse complaint ticket dedibox-console 
 dates:
-  validation: 2023-01-16
+  validation: 2023-07-24
   posted: 2020-03-12
 categories:
   - account
 ---
 
-An abuse ticket is a complaint about the practice of hosting copyrighted content or illegal behavior using Scaleway services.
+Abuse tickets are complaints regarding the hosting of copyrighted content or engaging in illegal activities using Scaleway services.
 
 <Message type="requirement">
-  - You have an account and are logged into the [Dedibox Console](https://console.online.net) 
+  - You have an account and are logged into the [Dedibox console](https://console.online.net) 
 </Message>
 
-1. Click your user name next to _logged in as:_ in the top right corner of the console. Then click **[Abuses](https://console-sudo.infra.online.net/en/account/abuses/list)** in the pop-up menu.
-    <Lightbox src="scaleway-dedibox-console-abuses-menu.webp" width="300" alt="" />
 
-  A list of your abuse tickets displays.
-2. Click **Details** to see the details of the complaint:
-    <Lightbox src="scaleway-dedibox-console-abuses-list.webp" width="800" alt="" />
-3. Take note of the details and complete the steps necessary to resolve the problem. You can find information on  how to resolve [outgoing DDoS](/compute/instances/reference-content/preventing-outgoing-ddos/) attacks from your server in our documentation.
-4. Describe the measures you have taken and provide an answer to the author of the ticket once the issue is solved. Click **Resolve** to mark the ticket as closed.
+1. Click your username, located next to "logged in as" at the top right corner of the Dedibox console. Then, select **Abuses** from the pop-up menu that appears.
+2. A list of your abuse tickets displays. Click **Details** to view the specifics of the complaint.
```

Comment:
This and step 4 do not start with a verb. I think it's best to send the first sentences back to the previous line.